src/main/kotlin/org/rust/ide/refactoring/RsRenameProcessor.kt

```diff
@@ -32,16 +33,19 @@ class RsRenameProcessor : RenamePsiElementProcessor() {
 
     override fun renameElement(element: PsiElement, newName: String, usages: Array<out UsageInfo>, listener: RefactoringElementListener?) {
         if (element is RsPatBinding) {
-            usages
-                .filter {
-                    val usageElement = it.element
-                    usageElement is RsStructLiteralField && usageElement.colon == null
+            usages.forEach {
+                val usageElement = it.element
+                if (usageElement is RsStructLiteralField && usageElement.colon == null) {
+                    val newPatField = RsPsiFactory(element.project).createStructLiteralField(element.text, newName)
+                    usageElement.replace(newPatField)
+                    return@forEach
                 }
-                .forEach {
-                    val newPatField = RsPsiFactory(element.project)
-                        .createStructLiteralField(element.text, newName)
-                    it.element!!.replace(newPatField)
+                val field = PsiTreeUtil.getParentOfType(usageElement, RsStructLiteralField::class.java)
```

Comment:
We usually use `ancestorStrict` or `ancestorOrSelf` instead of direct `PsiTreeUtil#getParentOfType`